src/main/kotlin/no/nav/omsorgspenger/routes/AaregRoute.kt

```diff
@@ -21,8 +23,13 @@ internal fun Route.AaregRoute(
         get {
             val stsToken = stsClient.token().asAuthoriationHeader()
             val path = call.request.uri.removePrefix("/aareg")
+            val personIdent = call.request.headers["Nav-Personident"]
+            val maskinportenToken = call.request.headers["Authorization"]
```

Comment:
Tokenet som kommer inn til proxyen er vel et azure-token, har ikke noe med maskinporten å gjøre?
Støtter aareg Azure-token siden du ønsker å sende det videre til aareg?

Skal ikke trenge å parse og legge til `Nav-Personident`, alle headere som sendes til proxyen sendes videre til den underliggende tjenesten. `extraHeaders` er kun om det er noen headere i tillegg som proxyen skal legge på, som f.eks. `Nav-Consumer-Token` her

Reply:
Tror det riktige blir at `extraHeaders` her kun er `HttpHeaders.Authorization` to stsToken og `Nav-Consumer-Token` to stsToken (begge til samme verdi)

Tilsvarende det som gjøres f.ekes. [her](https://github.com/navikt/helse-sparkelapper/blob/main/vilkarsproving/src/main/kotlin/no/nav/helse/sparkel/vilkarsproving/opptjening/AaregClient.kt#L25-L26)

Reply:
Aaa, da har jeg misforstått litt.
Er ikke OIDC-tokenet det refereres til i Swagger-dokumentasjonen en Maskinporten-token?
Hvis ikke, hvor skal det i såfall sendes?

Hvis headers blir videresent automatisk er jo den forsåvidt grei:)

Reply:
Så siste kommentar nå! Litt forvirrende at begge er STS-tokens, når man må registrere seg i Maskinporten.

Reply:
Det skal aldri sendes noe maskinporten-token til aareg, kun altinn-integrasjonen?

Det det referer til som OIDC token der er nok «gamle» innloggingsmekanismer - støtter ikke TokenX som vi skal bruke. Så her må vi bare kalle aareg som om vi er en tjeneste, ikke i context av en innlogget bruker.  Da skal begge headerne sendes som stsToken

Reply:
Den er grei! Skal jeg lage ny PR, eller får du lagt inn en endring med `Nav-Consumer-Token" to stsToken`?

Reply:
Du kan bare endre på denne branchen 👍